lib/rails_erd/diagram/graphviz.rb

```diff
@@ -310,7 +310,12 @@ def specialization_options(specialization)
       end
 
       def read_template(type)
-        ERB.new(File.read(File.expand_path("templates/#{NODE_LABEL_TEMPLATES[type]}", File.dirname(__FILE__))), nil, "<>")
+        template_text = File.read(File.expand_path("templates/#{NODE_LABEL_TEMPLATES[type]}", File.dirname(__FILE__)))
+        if ERB.instance_method(:initialize).parameters.assoc(:key) # Ruby 2.6+
```

Comment:
wouldn't it be easier and clearer to just use the RUBY_VERSION constant instead?

Reply:
This PR uses `ERB.instance_method(:initialize).parameters.assoc(:key)` instead of `RUBY_VERSION` of `ERB.new`. This approach is built into Ruby.
https://github.com/ruby/ruby/commit/3406c5d